docs/Migrating to version 2.0 guide.md

```diff
@@ -891,6 +1007,8 @@ now use the `overflowButton` slot:
 </ui5-tabcontainer>
 ```
 
+ - The display CSS property of the component is changed from `inline-block` to `block`.   
```

Comment:
I think we can remove this, it's not a breaking change like API

Reply:
Done.